python/setup.py

```diff
@@ -4,16 +4,14 @@
 from setuptools import setup
 from os import path
 
-PACKAGE_NAME = "ccftools"
+PACKAGE_NAME = "ccf"
 
-package_dir = path.join(path.abspath(path.dirname(__file__)), PACKAGE_NAME)
-print(package_dir)
-with open(path.join(package_dir, "README.md"), encoding="utf-8") as f:
+with open(path.join(path.abspath(path.dirname(__file__)), "README.md"), encoding="utf-8") as f:
     long_description = f.read()
 
 setup(
     name=PACKAGE_NAME,
-    version="0.11.10",
+    version="0.11.7",
```

Comment:
I remember trying and failing to get rid of this in cimetrics. I'll try to work it out this time, because I don't expect we'll be successful at not forgetting to update the version in the Python package.

Reply:
Yep. IIRC, the issue was that automated version detection based on the git tag would break outside of git. I think we could base it off cmake (see `version.cmake`) which handles both git and non-git repos.